### PR TITLE
changing naming to perftest

### PR DESCRIPTION
**Pull request by avi-hmcts**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Comment by thomast1906** [comment]
How does the plan look?

**Comment by avi-hmcts** [comment]
> How does the plan look?

seemed ok creating all the resources as expected. https://dev.azure.com/hmcts/CNP/_build/results?buildId=154108&view=logs&j=9529ead6-39ae-5319-1740-9d2035106bd6&t=89b2e481-5d95-5be1-d64b-e101e3d88784

File: pipeline-steps/deploy-network.yaml [hunk 1]
```diff
@@ -30,8 +30,10 @@ steps:
       workingDirectory: '$(System.DefaultWorkingDirectory)/components/${{ parameters.stack }}'
       backendType: azurerm
       backendServiceArm: ${{ parameters.terraformServiceConnection }}
-      backendAzureRmResourceGroupName: 'azure-control-$(env)-rg'
-      backendAzureRmResourceGroupLocation: ${{ parameters.location }}
+      ${{ if eq( parameters['env'], 'perftest') }}: 
+         backendAzureRmResourceGroupName: 'azure-control-test-rg'
+      ${{ if ne( parameters['env'], 'perftest' ) }}: 
+         backendAzureRmResourceGroupLocation: ${{ parameters.location }}
```
Comment (thomast1906) [inline pipeline-steps/deploy-network.yaml:36]: I don't think this is correct? We want to reference the `backendAzureRmResourceGroupName `only

**Review by thomast1906 — APPROVED** [review]

File: pipeline-steps/deploy-network.yaml (hunk 1 above)
Comment (avi-hmcts) [reply pipeline-steps/deploy-network.yaml]: This is resolved now

**Pull request merged**